### DISCUSSION TITLE
Inconsistency of alpha parameter in interval forecasts

### DISCUSSION THREAD

**Discussion by GrzegorzRut** [alan-turing-institute/sktime#2334] [Q&A]
I noticed that `AutoETS` and `AutoARIMA` have an opposite behavior when it comes to the predictions intervals specified by `alpha`.
I posted a question on [stack overflow](https://stackoverflow.com/questions/71545912/inconsistant-prediction-range-settings-in-sktime).

Is this the expected behavior? Is this due to 2 different libraries behind these algos?

**Comment by fkiraly** [comment]
It's a known (fixed) bug in `ARIMA` and conversions, where `alpha` was the wrong way round, more detail here:
https://github.com/alan-turing-institute/sktime/pull/2281

This should have been fixed with the 0.11.0 release, an upgrade should fix it for you.

**Reply by fkiraly** [reply]
(generally, intervals should get wider as `coverage` gets larger)

**Reply by fkiraly** [reply]
PS: the `return_pred_int` interface has been deprecated in 0.11.0, from now on it's `predict_interval` or `predict_quantiles`.

One of the main reasons we moved to the new interface was exactly this kind of confusion occurring frequently - between "interval coverage" (often called p), "one minus interval coverage" (often called alpha), "quantile value" (also often called alpha), and "quantile value of lower interval bound".
